src/sentry/static/sentry/app/views/organizationProjectsDashboard/teamSection.jsx

```diff
@@ -70,6 +70,7 @@ const TeamTitleBar = styled(Flex)`
 
 const TeamName = styled(PageHeading)`
   margin: 4px;
```

Comment:
I know you didn't introduce this, but while you're here can we kill this weird 4px margin on TeamName?

<img width="695" alt="Screen Shot 2019-05-13 at 5 38 18 PM" src="https://user-images.githubusercontent.com/30713/57662799-04610980-75a6-11e9-9bc1-4237cbd464f6.png">